services/121-service/src/payments/fsp-integration/excel/excel.service.ts

```diff
@@ -266,10 +331,7 @@ export class ExcelService
         fspConfig.id,
       );
     if (!registrationsForReconciliation?.length) {
```

Comment:
I think this `if` fits better in the end of the `getRegistrationsForReconciliation` method